readthedocs/rtd_tests/tests/test_search_json_parsing.py

```diff
@@ -19,6 +19,9 @@ def test_h2_parsing(self):
         )
         self.assertEqual(data['path'], 'api')
         self.assertEqual(data['sections'][1]['id'], 'a-basic-api-client-using-slumber')
+
+        # In api.fjson, title is in the form: A basic API client ``using slumber``
+        self.assertEqual(data['sections'][1]['title'], 'A basic API client using slumber')
```

Comment:
In my test the quotes will still be there:

    >>> pyquery.PyQuery('A basic API client ``using slumber``').text().replace('¶', '').strip()
    'A basic API client ``using slumber``'
    >>>

Reply:
When a text like this: `Include ``404`` page` is converted to html by sphinx. It is something like this: `Include <code class=\"docutils literal notranslate\"><span class=\"pre\">404</span></code> page`.

We were indexing this line as it is, because of which -- we are getting results with the html code - http://docs.celeryproject.org/en/latest/search.html?q=utilities&check_keywords=yes&area=default

It is something like this:
```python3
>>> import pyquery
>>> temp = 'Include <code class=\"docutils literal notranslate\"><span class=\"pre\">404</span></code> page'
>>> pyquery.PyQuery(temp).text().replace('¶', '').strip()
'Include 404 page' 
```

Reply:
Thanks for clarifying this!